docker/Dockerfile_aarch64_910B

```diff
@@ -0,0 +1,171 @@
+    make -j 16 && make install && \
+    ln -sf /usr/local/python${PYVERSION}/bin/python3 /usr/bin/python3 && \
+    ln -sf /usr/local/python${PYVERSION}/bin/python3 /usr/bin/python && \
+    ln -sf /usr/local/python${PYVERSION}/bin/pip3 /usr/bin/pip3 && \
+    ln -sf /usr/local/python${PYVERSION}/bin/pip3 /usr/bin/pip && \
+    cd .. && \
+    rm -rf Python* && \
+    mkdir -p ~/.pip  && \
+    echo '[global] \n\
+    index-url=http://mirrors.aliyun.com/pypi/simple\n\
+    trusted-host=mirrors.aliyun.com' >> ~/.pip/pip.conf && \
+    pip3 install pip -U
+
+RUN pip3 install -U pip && \
+    pip3 install wheel==0.43.0 scikit-build==0.18.0 numpy==1.24 setuptools==69.5.1 && \
+    pip3 install decorator sympy cffi && \
+    pip3 install cmake ninja pyyaml && \
+    pip3 install pathlib2 protobuf attrs attr scipy && \
+    pip3 install requests psutil absl-py && \
+    pip3 install torch==2.1.1 torchvision==0.16.1 --index-url=https://download.pytorch.org/whl/cpu && \
+    pip3 install transformers==4.38.0 && \
+    rm -rf /root/.cache/pip
+
+ENV LD_LIBRARY_PATH=/usr/lib/aarch64-linux-gnu/hdf5/serial:$LD_LIBRARY_PATH
+ENV LD_PRELOAD=/lib/aarch64-linux-gnu/libGLdispatch.so.0:$LD_PRELOAD
+
+RUN if [ ! -d "/lib64" ]; \
+    then \
+        mkdir /lib64 && ln -sf /lib/ld-linux-aarch64.so.1 /lib64/ld-linux-aarch64.so.1; \
+    fi
+
+FROM ubuntu:20.04 as buildtemp
+COPY ./*.run /tmp
+
+FROM export_image
+
+ENV LD_LIBRARY_PATH=\
+$ASCEND_BASE/driver/lib64:\
+$ASCEND_BASE/driver/lib64/common:\
+$ASCEND_BASE/driver/lib64/driver:\
+$ASCEND_BASE/driver/tools/hccn_tool/:\
+$TOOLKIT_PATH/opp/built-in/op_impl/ai_core/tbe/op_tiling/lib/linux/aarch64/:\
+$LD_LIBRARY_PATH
+
+RUN --mount=type=cache,target=/tmp,from=buildtemp,source=/tmp \
+    umask 0022 && \
+    mkdir -p $ASCEND_BASE/driver && \
+    if [ "$CHIP" != "all" ]; \
+    then \
+        CHIPOPTION="--chip=$CHIP"; \
+    else \
+        CHIPOPTION=""; \
+    fi && \
+    chmod +x $TOOLKIT_PKG $KERNELS_PKG && \
+    ./$TOOLKIT_PKG --quiet --install --install-path=$ASCEND_BASE --install-for-all $CHIPOPTION && \
+    ./$KERNELS_PKG --quiet --install --install-path=$ASCEND_BASE --install-for-all && \
+    rm -f $TOOLKIT_PKG $KERNELS_PKG
+
+ENV GLOG_v=2 \
+    LD_LIBRARY_PATH=$TOOLKIT_PATH/lib64:$LD_LIBRARY_PATH \
+    TBE_IMPL_PATH=$TOOLKIT_PATH/opp/op_impl/built-in/ai_core/tbe \
+    PATH=$TOOLKIT_PATH/ccec_compiler/bin:$PATH \
+    ASCEND_OPP_PATH=$TOOLKIT_PATH/opp \
+    ASCEND_AICPU_PATH=$TOOLKIT_PATH
+
+ENV PYTHONPATH=$TBE_IMPL_PATH:$PYTHONPATH
+
+RUN rm -rf ./*
+
+SHELL ["/bin/bash", "-c"]
+RUN echo "source /usr/local/Ascend/ascend-toolkit/set_env.sh" >> ~/.bashrc && \
+    . ~/.bashrc
+
+WORKDIR /deeplink
+RUN echo -e "diff --git a/impl/ascend_npu/CMakeLists.txt b/impl/ascend_npu/CMakeLists.txt\n\
+index e684c59..f1cd8d4 100755\n\
+--- a/impl/ascend_npu/CMakeLists.txt\n\
++++ b/impl/ascend_npu/CMakeLists.txt\n\
```

Comment:
What's this for?

Reply:
Warning patch, as you mentioned before such warnings can be misunderstood by users as errors.